interfaces/cython/cantera/examples/onedim/flame_initial_guess.py

```diff
@@ -0,0 +1,164 @@
+"""
+A freely-propagating, premixed methane-air flame. 
+Examples of saving and loading a flame and restarting 
+with different initial guesses.
+
+Requires: cantera >= 2.6.1
```

Comment:
```suggestion
Requires: cantera >= 3.0
```
I hope we can avoid 2.6.1